src/gui/Src/BasicView/Disassembly.cpp

```diff
@@ -116,6 +116,8 @@ void Disassembly::updateColors()
 void Disassembly::updateFonts()
 {
     setFont(ConfigFont("Disassembly"));
+    delete mFontMetrics;

```

Comment:
Yes that's the whole point of virtual functions. If you call the base class
function you don't need to duplicate the delete mFontMetrics line (actually
not sure why I don't call the base class here).
On Mon, 6 Jun 2016 at 10:44, Torusrxxx <notifications@github.com> wrote:

> In src/gui/Src/BasicView/Disassembly.cpp
> <https://github.com/x64dbg/x64dbg/pull/723#discussion_r65854843>:
>
> > @@ -116,6 +116,8 @@ void Disassembly::updateColors()
> >  void Disassembly::updateFonts()
> >  {
> >      setFont(ConfigFont("Disassembly"));
> > +    delete mFontMetrics;
>
> No, updateFonts is a virtual function which is already overridden.
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/x64dbg/x64dbg/pull/723/files/509a0f6ba42a1e53b3f45f8a2a124b2d0bfb4742#r65854843>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ACWCmdvQrtpFHhCsee7xX-dhWO3WUyB1ks5qI93lgaJpZM4Iukas>
> .
>